src/transactions/test/TxEnvelopeTests.cpp

```diff
@@ -2377,12 +2377,12 @@ TEST_CASE_VERSIONS("txenvelope", "[tx][envelope]")
 
                     auto r = closeLedgerOn(*app, 1, 2, 2016, {tx1, tx2});
 
-                    REQUIRE(tx1->getResultCode() == txSUCCESS);
-                    REQUIRE(tx2->getResultCode() == txFAILED);
+                    checkTx(0, r, txSUCCESS);
```

Comment:
I think you need to pass `true` to the last arg of `closeLedgerOn`, otherwise you don't know if `tx1 < tx2`

Reply:
I don't think so: results are in the apply order (I've double-checked with the debugger). The last arg of `closeLedgerOn` makes the txs to be applied in the same order as we have passed to the method (which already happens due to seq nums) and it has nothing to do with the result order.

Reply:
Ah didn't see that tx1 and tx2 were from the same account!